src/planar/postprocessors.jl

```diff
@@ -88,16 +82,24 @@ function _insert_planar_operations(ex)
     return ex
 end
 
-# Mimick `TO.insert_operationbackend` for planar operations.
-function insert_operationbackend(ex, backend)
-    if isexpr(ex, :call) && ex.args[1] isa GlobalRef &&
-       ex.args[1].mod == TensorKit &&
-       ex.args[1].name ∈ _PLANAR_OPERATIONS
-        b = Backend{backend}()
-        return Expr(:call, ex.args..., b)
-    elseif isa(ex, Expr)
-        return Expr(ex.head, (insert_operationbackend(e, backend) for e in ex.args)...)
-    else
-        return ex
-    end
+"""
```

Comment:
These are not yet used currently? This is in preparation of further changes to the planar parser?